site/content/includes/m3aggregator/annotated_config.yaml

```diff
@@ -0,0 +1,754 @@
+# Logging configuration
+# TODO: More detail than this 
+# https://github.com/m3db/m3/blob/9f129cf9f16430cc5a399f60aa5684fb72b55bb5/src/cmd/services/m3query/config/config.go#L116
+# Configuration for logging.
+logging:
+  # Log file location
+  file: <string>
+  # Error logging level
+  level: <string>
+  # TODO: ??
+  fields:
```

Comment:
Perhaps just remove these?

Reply:
@robskillington I will, but at the moment, there are also placeholders of things to check with you when we review fully ;)